docs/REDUX_STRUCTURE.md

```diff
@@ -0,0 +1,120 @@
+# Redux Multi Currencies Structure
+
+## Store structure
+For being able to store multi currencies data, we should need a store structure more similar to the one at [lisk-mobile](https://github.com/liskHQ/lisk-mobile).
+
+### Lisk Mobile
+Showing here just the parts related to the multi currencies structure.
+```json
+// lisk-mobile store example
+{
+  "..."
+  "accounts": {
+    "passphrase": "",
+    "followed": { "tokenKey": [{}] },
+    "info": { "tokenKey": { "address": "", "balance": "", "publickey": "", "..." } }
+  },
+  "service": {},
+  "settings": {
+    "token": {
+      "active": "tokenKey",
+      "list": { "tokenKey": true }
+    },
+    "..."
+  }
+}
+```
+Where we have the accounts information and the followed accounts separated per token type, and other informations that are relevant for more than one token being directly on the `accounts` node.  
+On `settings` we have which token is the active one, and also a list with the tokens, of which token is enabled by the user.  
+And also a unified `service` node, not just as liskService as we have right now on Lisk-Hub.  
+
+### Lisk Hub
+In the case of Lisk-Hub we could have initialy a structure that integrate some of the changes just for other currencies, while we don't update the current LSK structure.  
+To have a better structure for new currencies, while also keeping the LSK token working without too much work, something like:
+```json
+// lisk-hub intermediate store example
+{
+  "..."
+  "account": { "LSK account info" },
+  "accounts": { "info": { "BTC": {} }, "followed": {} },
+  "BTC": { "BTC specific data" },
+  "service": {},
```

Comment:
This reducer needs to support multi token. the structure of this reducer should be similar accounts reduce. Also, lets the app to store dynamic fee system for BTC.

Reply:
the service data structure should be like the below example :
 service: { dynamicFees: {} , priceTicker: { "BTC": {}, "LSK": {} }};